nemo/collections/common/metrics/perf_metrics.py

```diff
@@ -33,6 +40,16 @@ class FLOPsMeasurementCallback(Callback):
 
     higher_is_better = True
 
+    def __new__(cls, model_config, *args, **kwargs):
+        create_tensorboard_logger = model_config.get('exp_manager', {}).get("create_tensorboard_logger", False)
```

Comment:
#Supports tensorboard logger today. When new loggers are supported, add the check here.